src/php/utils/docker/docker-php-dev-mode

```diff
@@ -29,7 +29,7 @@ case "$1" in
 			exit 1
 		fi
 
-		pecl install xdebug-stable
+		pecl install xdebug-2.9.8
```

Comment:
No change `xdebug-2` is possible here? Doubt it but would be neat

Reply:
I tried it when the Xdebug 3 first hit the beta channel, but didn't work. Hence pinning it here to a specific version. In any case, this solution is temporary until we do the changes needed to adopt Xdebug 3, which aren't many as far as my initial research went.

Reply:
O well was worth a shot 👍